01 - intro.md

```diff
@@ -19,12 +19,27 @@ them. At the same time, we want to get out of the business of procuring,
 maintaining, or even _thinking_ about hardware, so all of the above happens in
 hosted infrastructure run by multiple cloud vendors.
 
-<!---
+## What Is Cloud-Native?
```

Comment:
I ended up completely missing this review and had it on my list to do my own take of this, which I did in #36 ... I'll let you decide what to take/leave.